src/ArcgisAccount/ArcgisAccount.js

```diff
@@ -122,6 +123,7 @@ class ArcgisAccount extends Component {
         onRequestClose={this.closeAccountControl}
         placement="bottom-end"
         positionFixed
+        appendToBody={appendToBody === undefined ? true : appendToBody}
```

Comment:
This ternary can be avoided if we add `appendToBody` to the props definitions at the end of the file (propTypes and defaultProps). This will also add the property to the table in our docs.